docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites.md

```diff
@@ -42,6 +34,20 @@ A node running YugabyteDB Anywhere is expected to meet the following requirement
 - 100 GB SSD disk or more
 - 64-bit CPU architecture
 
+### Kubernetes-based installations
+
+A Kubernetes node is expected to meet the following requirements: 
+
+- 5 cores (minimum) or 8 cores (recommended)
+
+- 15 GB RAM (minimum)
+
+- 250 GB SSD disk (minimum) allocated to YugabyteDB Anywhere 
+
+  Since resizing StatefulSets after their creation is challenging, it is recommended to plan for long-term storage needs and allocate more storage initially.
```

Comment:
This comment is technically out of place because it is about the storage class volume, not the node storage. I made a mistake in the JIRA ticket - let's remove this for now and we can figure out the right callout for this, maybe in this section 

https://deploy-preview-15391--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes/#customize-yugabytedb-anywhere

Reply:
Removed this sentence